service/history/workflow_task_handler.go

```diff
@@ -345,6 +349,15 @@ func (handler *workflowTaskHandlerImpl) handleCommand(
 		}
```

Comment:
Looks like we have a command registery as well where we dynamically register and handle commands on line 337 above. `ch, ok := handler.commandHandlerRegistry.Handler(command.GetCommandType())`

There are currently just 2 commands being registered dynamically (`COMMAND_TYPE_SCHEDULE_NEXUS_OPERATION` & `COMMAND_TYPE_REQUEST_CANCEL_NEXUS_OPERATION`). Both of these create history events. Do we care about user metadata in these commands (or any that may be registered in the future). cc @yiminc @cretz

Reply:
@bergundy thoughts?

Reply:
Unless it's a real problem, I think we should consistently copy all command metadata to all related event metadata regardless of the internal code paths.

Reply:
This would require a larger refactor. Can we do it if actually required?